server/users/routes.js

```diff
@@ -17,6 +17,19 @@ userRouter.get('/', authMiddleware, async (req, res) => {
     });
 });
 
+// GET users
+userRouter.get('/:id', async (req, res) => {
+  const { id } = req.params;
+  await getUserById(id)
+    .then((user) => {
+      res.status(200).send(user);
+    })
+    .catch((err) => {
+      console.log(err);
+      res.status(500).json({ error: 'something went wrong retreving users from db.' });
+    });
+});
```

Comment:
Can we either use `await` or use `.then()`, not both?

Reply:
Done. I went with await. I guess I should change all the other routes too? 
![image](https://user-images.githubusercontent.com/35232697/107993574-5cf5df80-6fa0-11eb-8998-7d7a9cd817c1.png)

Reply:
Thanks! Preferably, but usually it's fine to limit the scope of things you touch

Reply:
I changed all the ones in `users`